src/structures/VoiceState.js

```diff
@@ -63,10 +64,22 @@ class VoiceState extends Base {
      */
     this.streaming = data.self_stream || false;
     /**
-     * The ID of the voice channel that this member is in
+     * The ID of the voice channel or stage channel that this member is in
      * @type {?Snowflake}
      */
     this.channelID = data.channel_id || null;
```

Comment:
For some reason `channelID` is null for my VoiceState when I'm in a stage channel. I haven't found a way to reproduce yet.

Reply:
Unfortunately I can't reproduce this - it would be great if you could give some more information on when/how this occurs, and a code snippet would help too!

Reply:
was caused by not setting the `GUILD_VOICE_STATES` intent